README.md

```diff
@@ -301,6 +301,24 @@ Notes are sorted by "relevance". Currently it means the notes that contain the r
 
 In the desktop application, press Ctrl+G or Cmd+G and type the title of a note to jump directly to it. You can also type `#` followed by a tag or `@` followed by a notebook title.
 
+# Privacy
+
+Joplin values your privacy and security by giving you complete control over your information and digital footprint.
+
+Joplin applications do not send any data to any service without your authorisation. Any data that Joplin saves, such as notes or images, are saved to your own device and you are free to delete this data at any time.
+
+Joplin has many modern features, some of which use third-party services. You can be completely disable any or all of these features in the application settings. These features are:
```

Comment:
"You can completely disable..."

Reply:
Done.  I also removed the "completely" as it doesn't add anything to the meaning.